private/bufpkg/bufmodule/bufmoduleapi/registry.go

```diff
@@ -125,7 +125,7 @@ func getSingleRegistryForContentModules(contentModules []bufmodule.Module) (stri
 	for _, module := range contentModules {
 		moduleFullName := module.ModuleFullName()
 		if moduleFullName == nil {
-			return "", syserror.Newf("expected module name for %q", module.OpaqueID())
+			return "", syserror.Newf("expected module name for %q", module.Description())
```

Comment:
Description vs opaqueID does not really matter for syserrors, but I changed it just in case.